src/antsibull/data/ansible-setup_py.j2

```diff
@@ -206,9 +206,14 @@ setup(
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
 {%- endif %}
+{%- if version.major >= 5 and version.major < 7 %}
         'Programming Language :: Python :: 3.8',
+{%- endif %}
         'Programming Language :: Python :: 3.9',
         'Programming Language :: Python :: 3.10',
+{%- if version.major >= 7 %}
+        'Programming Language :: Python :: 3.11',
+{%- endif %}
```

Comment:
Maybe we should simply copy the values over from ansible-core? Might be easier than having specialized code which "magically" creates this.